[2025-01-13T17:11:34.248+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: finflow_source_data_pipeline.generate_account_data manual__2025-01-13T17:11:10.807825+00:00 [queued]>
[2025-01-13T17:11:34.267+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: finflow_source_data_pipeline.generate_account_data manual__2025-01-13T17:11:10.807825+00:00 [queued]>
[2025-01-13T17:11:34.268+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-01-13T17:11:34.298+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): generate_account_data> on 2025-01-13 17:11:10.807825+00:00
[2025-01-13T17:11:34.310+0000] {standard_task_runner.py:57} INFO - Started process 8767 to run task
[2025-01-13T17:11:34.325+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'finflow_source_data_pipeline', 'generate_account_data', 'manual__2025-01-13T17:11:10.807825+00:00', '--job-id', '424', '--raw', '--subdir', 'DAGS_FOLDER/finflow.py', '--cfg-path', '/tmp/tmpa1qwdhp3']
[2025-01-13T17:11:34.332+0000] {standard_task_runner.py:85} INFO - Job 424: Subtask generate_account_data
[2025-01-13T17:11:34.450+0000] {task_command.py:416} INFO - Running <TaskInstance: finflow_source_data_pipeline.generate_account_data manual__2025-01-13T17:11:10.807825+00:00 [running]> on host ab4e86dc450e
[2025-01-13T17:11:34.627+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='finflow_source_data_pipeline' AIRFLOW_CTX_TASK_ID='generate_account_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-13T17:11:10.807825+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-13T17:11:10.807825+00:00'
[2025-01-13T17:11:34.630+0000] {finflow.py:76} INFO - Starting data generation for generate_account_data
[2025-01-13T17:11:34.632+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:314: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2025-01-13T17:11:34.878+0000] {finflow.py:78} INFO - Successfully completed data generation for generate_account_data
[2025-01-13T17:11:34.878+0000] {python.py:194} INFO - Done. Returned value was: /tmp/tmpfju0rjl0/accounts.parquet
[2025-01-13T17:11:34.918+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=finflow_source_data_pipeline, task_id=generate_account_data, execution_date=20250113T171110, start_date=20250113T171134, end_date=20250113T171134
[2025-01-13T17:11:34.987+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-13T17:11:35.077+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
